packages/lib/src/components/Card/types.ts

```diff
@@ -100,9 +101,35 @@ export interface CardElementProps extends UIElementProps {
 
 export type SocialSecurityMode = 'show' | 'hide' | 'auto';
 
+// TODO clarify exact properties that can be in this object
+//  - should only be ones that can be sent in the configuration object in the /paymentMethods response
+/** If the merchant wishes to set any of these properties in their local config they should do so via a "configuration" object */
 export interface CardConfiguration {
+    // GooglePay
+    merchantIdentifier?: string;
+    merchantOrigin?: string;
+    gatewayMerchantId?: string;
+
+    // AmazonPay
+    publicKeyId?: string;
+    region?: string;
+
+    // Common to GooglePay & ApplePay
+    merchantName?: string;
+
+    // Common to GooglePay & AmazonPay
+    merchantId?: string;
+
+    // Paypal
+    intent?: string;
```

Comment:
Maybe we should extract this ones to either a global or more specific type

Reply:
This was really just an early attempt to clarify what a `CardConfiguration` object looks like.
If you think it needs more work I'll move it to another PR - since I was just tacking it on to this "error message" PR